### PR TITLE
Update README.md to render / read correctly

### PR DESCRIPTION
**Pull request by willend**
Fix various typos / .md string import issues to let README.md show the actual project titles

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ## Student projects at [European Spallation Source Data Management and Software Centre](https://europeanspallationsource.se/data-management-software-centre)
 * [Help us improve support for e.g. STEP or STL geometries](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_CAD.pdf)
 * [Help us improve 3D visualisa0on in McStas](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_CAD_mcdisplay.pdf)
-* [Use MCPL and KDSource u3ls and ML / AI to generate “analy3cal” McStas source models from par3cle clouds](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_KDSource.pdf)
+* [Use MCPL and KDSource utils and ML / AI to generate “analytical” McStas source models from particle clouds](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_KDSource.pdf)
```
Comment (nvaytet) [inline README.md]:
```suggestion
* [Use MCPL and KDSource utils and ML / AI to generate "analytical" McStas source models from particle clouds](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_KDSource.pdf)
```

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 * [Numerical optimization of Monte-Carlo ray-tracing simulation of neutron scattering instrumentation using McStas and McStasScript Python API](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_Optimizer.docx)
-* [Projects within SasView – a toolkit for analysing Small Angle Sca9ering data](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_SasView.pdf)
+* [Projects within SasView – a toolkit for analysing Small Angle Scattering data](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_SasView.pdf)
 * [Improving variance reduction in McStas Union components](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_Union_variance.docx)
 * [Optimization of neutron guides for very long wavelengths that counteract the vertical aberrations from gravity](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_VCN_guide_bot.docx)
 * [Implement an image-recogni0on based data quality monitor](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_ai_detector_alarms.pdf)
```
Comment (nvaytet) [inline README.md]:
```suggestion
* [Implement an image-recognition based data quality monitor](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_ai_detector_alarms.pdf)
```

File: README.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 * [Creating knowledge graph and estimating biases in data from photon and neutron open science cloud data portal](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_federated_search_knowledge_graph.pdf)
-* [Modify exits,ng McStas op,cal component models to allow “reverse” raytracing from “sample” to “source”](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_reverse_Optics.pdf)
+* [Modify exitsting McStas optical component models to allow “reverse” raytracing from “sample” to “source”](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_reverse_Optics.pdf)
```
Comment (nvaytet) [inline README.md]:
```suggestion
* [Modify existing McStas optical component models to allow "reverse" raytracing from "sample" to "source"](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_reverse_Optics.pdf)
```

File: README.md [hunk 4]
```diff
@@ -1,14 +1,14 @@
 ## Student projects at [European Spallation Source Data Management and Software Centre](https://europeanspallationsource.se/data-management-software-centre)
 * [Help us improve support for e.g. STEP or STL geometries](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_CAD.pdf)
 * [Help us improve 3D visualisa0on in McStas](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_CAD_mcdisplay.pdf)
```
Comment (nvaytet) [inline README.md]:
```suggestion
* [Help us improve 3D visualisation in McStas](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_CAD_mcdisplay.pdf)
```

**Review by nvaytet — APPROVED** [review]

**Comment by willend** [comment]
Thanks @nvaytet - and sorry for having overlooked the review for ages :-D

**Pull request merged**